hail_search/hail_search_query.py

```diff
@@ -22,6 +23,15 @@ def _to_camel_case(snake_case_str):
 
 class BaseHailTableQuery(object):
 
+    GENOTYPE_QUERY_MAP = {
+        REF_REF: lambda gt: gt.is_hom_ref(),
+        REF_ALT: lambda gt: gt.is_het(),
```

Comment:
Would you mind giving some context about why both `REF_ALT` and `COMP_HET_ALT` are both `gt.is_het()`?  I might have expected one `is_het_non_ref` and one `is_het_ref`.

Reply:
for gCNV the behavior is different for `COMP_HET_ALT` and its overridden for that class. But if its confusing I can take this out for now and add it back when I actually add gCNV support